py/makeqstrdefs.py

```diff
@@ -20,6 +21,11 @@
 # Extract MP_COMPRESSED_ROM_TEXT("") macros.  (Which come from MP_ERROR_TEXT)
 _MODE_COMPRESS = "compress"
 
+# How many concurrent pre-processor instances to run to extract QSTRs.
+_PREPROCESS_PARALLELISM = 8
+# How many files to pre-process at a time.
+_PREPROCESS_BATCH_SIZE = 20
```

Comment:
Why not just do the maximum, ie each process has `TOTAL_FILES / NUM_PROCESS`?

Reply:
Done